model/access_control_entry.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require_relative "../model"
+
+class AccessControlEntry < Sequel::Model
```

Comment:
The reason for the combined commit is the ubid code is broken if you introduce the `TYPE_*` constants before the models, and in order to generate the static ubids in the migration, the `TYPE_*`  constants were added. It looks like the vanity ubid code doesn't need the type constant, so it's possible that commit could be split, even though it would look strange as the ubids generated in the migration would be using prefixes not yet added to the ubid code.  Do you want me to split that commit?

Reply:
Will this code fail when we deploy it, until we run the migration? If it will fail, yes we should split it

Reply:
OK.  I'll split the migration file into a separate commit.